templates/cluster-api/values.yaml

```diff
@@ -1,11 +1 @@
-manager:
-  image:
-    repository: registry.k8s.io/cluster-api
-    name: cluster-api-controller
-    tag: v1.7.2
-  imagePullPolicy: IfNotPresent
-
-clusterTopology: false
-
-expClusterResourceSet: false
-expMachinePool: false
+version: v1.7.2
```

Comment:
I'm not sure whether we want to make it configurable here or make it strictly dependent on the HMC version @Schnitzel

Reply:
I'm ok to have the CAPI version bound to a specific 2A version and NOT have it configurable, as in the end there is so many ways that CAPI and 2A could be out of sync if we have too much version skew.